packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/TxDetailModal/BasicTxDetails.tsx

```diff
@@ -140,12 +140,22 @@ export const BasicTxDetails = ({
                 {/* TX ID */}
                 <Item label={<Translation id="TR_TXID" />} iconName="biometric">
                     <IOAddress
-                        txAddress={tx.txid}
+                        txAddress={tx.symbol === 'xlm' ? tx.txid.split('#')[0] : tx.txid}
```

Comment:
usually we do `networkType` in case we add stellar testnet

Reply:
Previously, we displayed a single Operation, and the txid might be duplicated, so this was necessary. Now we are displaying a single tx, so the txid is unique, and we no longer need to perform this magic.